src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -298,14 +298,16 @@ function YieldCard({
           return (
             <View style={styles.cardLineContainer} key={index}>
               <View style={styles.cardLineLabel}>
-                <Text numberOfLines={1} style={styles.cardLabelText}>
-                  {rate.label}
-                </Text>
-                <TokenIcons
-                  tokensInfo={tokenInfo}
-                  size={IconSize.XXSMALL}
-                  showNetworkIcon={false}
-                />
+                <View style={styles.yieldRateLabelContainer}>
```

Comment:
All of the other contents of a `<View style={styles.cardLineLabel}>` are a single component, and it makes the styles simpler if this one is too. So, I added a wrapper component